project/Settings.scala

```diff
@@ -44,6 +46,15 @@ object Settings extends Build {
     versionStatus        := Versions.status(scalaVersion.value, scalaBinaryVersion.value)
   )
 
+  lazy val sparkPackageSettings = Seq(
```

Comment:
@brkyvz Doing that would mean you want everything to be under assembly, and we do not.